scripts/mar_to_map.py

```diff
@@ -0,0 +1,42 @@
+#!/bin/python3
+
+import json, os, sys
+
+def main(args):
+    try:
+        path = args[1]
+        outPath = args[2]
+    except IndexError:
+        sys.exit(f"Usage: {args[0]} .mar .bin")
+
+    jsonPath  = path.replace(".mar", ".json")
+    if not os.path.exists(jsonPath):
+        raise FileNotFoundError("Map layout info not found: "+jsonPath)
+
+    with open(jsonPath, "r", encoding="utf-8") as f:
+        data = json.load(f)
+    with open(path, "rb") as f:
+        marData = f.read()    
+
+    out = bytearray()
+    out.append(data["width"])
+    out.append(data["height"])
+    out.extend(convert_from_mar(marData))    
+
+    with open(outPath, "wb+") as f:
+        f.write(out)
+
+def convert_from_mar(data):
+    # based on: https://github.com/FEBuilderGBA/FEBuilderGBA/blob/master/FEBuilderGBA/MapEditorForm.cs#L1293
+    out = bytearray()
+    for i in range(0, len(data), 2):
+        value = int.from_bytes([data[i+1],data[i] ])
```

Comment:
When I run this on my WSL setup I'm getting the following:

```
scripts/mar_to_map.py  graphics/map/layout/AnInterestingMap.mar graphics/map/layout/AnInterestingMap.bin
Traceback (most recent call last):
  File "<...>/fireemblem8u/scripts/mar_to_map.py", line 42, in <module>
    main(sys.argv)
  File "<...>/fireemblem8u/scripts/mar_to_map.py", line 24, in main
    out.extend(convert_from_mar(marData))
  File "<...>/fireemblem8u/scripts/mar_to_map.py", line 33, in convert_from_mar
    value = int.from_bytes([data[i+1],data[i] ])
TypeError: from_bytes() missing required argument 'byteorder' (pos 2)
make: *** [Makefile:172: graphics/map/layout/AnInterestingMap.bin] Error 1
```

My Python version is:

```
python3 --version
Python 3.10.6
```

Reply:
I also had an issue where I had to `chmod +x` the script in order for it to run the first time:

```
scripts/mar_to_map.py  graphics/map/layout/AnInterestingMap.mar graphics/map/layout/AnInterestingMap.bin
make: scripts/mar_to_map.py: Permission denied
make: *** [Makefile:172: graphics/map/layout/AnInterestingMap.bin] Error 127
```